modules/hcp-ibm-power-create-heterogeneous-nodepools-agent-hc.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-create-heterogeneous-nodepools-agent-hc_{context}"]
+= Creating heterogeneous node pools on agent hosted clusters
+
+You can create heterogeneous node pools on the agent platform. It enables clusters to run diverse machine types, for example, `x86_64` or `ppc64le`, within a single hosted cluster. 
+
+To create a heterogeneous node pool, you must create an agent service configuration custom resource (CR) that informs the Operator how much storage is needed for components such as the database and filesystem. The CR also defines which {product-title} versions to maintain.
```

Comment:
If it's not too much work, maybe do something like this. 

Make this module a CONCEPT and list the general steps, represented by the other modules. Make the procedure its own module. Does that make sense? 

```
To create a heterogeneous node pool, perform the following general steps, as described in the following sections: 
* Create an agent service configuration custom resource (CR).
* Create an agent cluster.
* Create the heterogeneous node pool.
* Configure DNS for hosted control planes
* Create an `infraenv` custom resource (CR) for each architecture.
* Add agents to the heterogeneous cluster.
```
If any of these steps is optional, you could indicate it in the list to be clear, for example: 

* Optional: Create the heterogeneous node pool.

Reply:
@mburke5678 I have updated the PR as per above.
```
$ git status
On branch hcp_update
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        new file:   modules/hcp-ibm-power-create-heterogeneous-nodepools-agent-hc-con.adoc
        modified:   modules/hcp-ibm-power-create-heterogeneous-nodepools-agent-hc.adoc
```
Request you to take a look. Thanks.